app/views/obituaries/covid/index.html.erb

```diff
@@ -13,4 +13,24 @@
   </div>
   <hr>
 <% end %>
+
+  <% @obituaries_api.each do |obit| %>
```

Comment:
These is where the styling comes in

Reply:
Nice! Maybe I can take this off your plate so you can finish building out search (if you aren't sick of it).